content/pages/volunteers.md

```diff
@@ -71,17 +76,6 @@ Here are the volunteer positions currently available.
   positions. In addition to attending the event, you will also have to spend
   time familiarizing yourself with the material prior to it.
 
-### Systems Committee Volunteer
-
-* This is a great way to learn about Linux systems administration, version
-  control, and static website development!
-* You’ll work with our WiCS systems committee and help maintain our machines,
-  track issues, etc.  Indicate your interest in being a systems-committee
-  volunteer in the form above. Doing so will add you to the wics-sys mailing list.
-* **You will need:** to be willing to work independently, try new things,
-  follow along with existing work, and ask questions. Prior experience, although
-  appreciated, is not a pre-requisite.
-
 ### Run an event with us!
```

Comment:
Maybe we should move this up to the top of the page? Currently this is the only volunteer "position" we can support.